backend/src/main/java/corea/room/service/RoomService.java

```diff
@@ -48,10 +51,8 @@ public RoomResponse create(long memberId, RoomCreateRequest request) {
                 .orElseThrow(() -> new CoreaException(ExceptionType.MEMBER_NOT_FOUND, String.format("%d에 해당하는 멤버가 없습니다.", memberId)));
         Room room = roomRepository.save(request.toEntity(member));
 
-        long roomId = room.getId();
         participationRepository.save(new Participation(room, memberId));
-        automaticMatchingRepository.save(new AutomaticMatching(roomId, request.recruitmentDeadline()));
-        return RoomResponse.of(room, true);
+        return RoomResponse.of(room, ParticipationStatus.PARTICIPATED);
```

Comment:
```suggestion
        return RoomResponse.of(room, ParticipationStatus.MANAGER);
```

아무래도 의도하는바가 `MANAGER`가 맞겠지요?